product-service/src/handlers/getProductsById/index.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import getCors from '../../utils/getCors';
+import { getById } from '../../services/productService';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+  const headers = getCors();
+
+  try {
+    const { productId } = event.pathParameters;
+    const product = await getById(productId);
+
+    if (!product) {
+      return {
+        statusCode: 400,
```

Comment:
I think using 404 status code will be more correctly in case of source not found